manifests/e/ElectronicTeam/VirtualSerialPortDriver/11.0.1047/ElectronicTeam.VirtualSerialPortDriver.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ElectronicTeam.VirtualSerialPortDriver
+PackageVersion: 11.0.1047
+InstallerType: inno
+Installers:
```

Comment:
The VSPD installer contains both x64 and x86 build, autodetects the system architecture and installs the correct build accordingly.